src/job_description_feature_extraction.py

```diff
@@ -0,0 +1,49 @@
+import string
+import operator
+from collections import Counter
+
+
+def get_one_hot_encoded_words(feature_to_extract):
+    """
+    get_feature_word_count(x) returns a list of tuples which consists of word:count
+    From those lists, the words appear in the list below ( word_list ) was hand picked
+    Fhey are the word which might have an impact on the salary
+    """
+    word_list = ['excellent', 'graduate', 'immediate', 'junior', 'urgent']
```

Comment:
Just because a word occurred many times doesn't mean it has an effect on the prediction. Just run the script and get 200ish comments. Most of them are bullshit
teacher,team,accountant, uk, up, well, web, west, east, who, will, with, within, working, worker are some examples :smile: